### ISSUE TITLE
Python 3 compatibility

### ISSUE THREAD

**Issue by abulka** [theojulienne/PySignals#6]
Looks like this library is not compatible with Python 3 e.g.

```python
import pysignals

pizza_done = pysignals.Signal()

pizza_done.send(sender='fred', toppings=123, size=56)
```

gives the error

```
File "/Users/xxx/.pyenv/versions/3.9.9/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/pysignals/dispatcher.py", line 233
    except Exception, err:
                    ^
SyntaxError: invalid syntax
```

**Comment by knight-of-ni** [comment]
That error is referring to the previous version of pysignals and has been fixed in 0.1.3

**Comment by knight-of-ni** [comment]
I'm guessing this error was caused due to an outdated version of pysignals on PyPi. Since we have an open issue #5 for that, I'm going to this issue out.

Feel free to respond if this is not the case.
